docs/curation-guidelines.md

```diff
@@ -63,23 +63,41 @@ To evaluate the discovered licenses for a package component, use the following s
     1. If there is/are SPDX-identified license(s) in the file, use the SPDX license expression to capture those licenses.
     2. If there is no license in the file, code `NONE`.
     3. If there is a license, but that license does not have an SPDX identifier, code `OTHER`.
-3. If the definition indicates a source location, curate the license information found on source component’s definition. Because of the flow through of license obligations from source to a binary package, the information from the source location’s core facet should be encoded as a “discovered license” in the package.
+3. Follow the above steps for the source component definition that corresponds to the package component definition.  If you see `</>` next to the magnifying glass symbol, you can pull it up automatically. 
 
-## Troubleshooting: sources of truth for the declared license
+### Canonical locations for the declared license for source definitions 
+When curating the declared license for a source code repository, look in the canonical locations listed below.  
+1.	**License file.**  Look to see if a license file (e.g. LICENSE, COPYING) is included in the specific commit or version you are curating.  Sometimes the license is found on the README.  There may be more than one declared license.  If there is not any license information at the specific commit or version level, check the master branch.
+2.	**File Headers.**  If there is not a declared license anywhere in the repository, look to see if there is any license information in the file headers.  Look at multiple files to determine if they are under different licenses.  You can also try using a search function to find any mention of ‘license,’ for example.
```

Comment:
Rather than ...`look to see if there is any license information in the file headers`... what about talking about ...`look to see if there is any license information in the source code file header comments`...

Reply:
For me at least, both variations of the wording effectively communicate the same action (to examine the file's contents for any header text that may tell us what license is applicable to it), but I guess the more specific we are the better.